package.json

```diff
@@ -158,6 +159,7 @@
     "sass": "^1.22.10",
     "semver": "^6.3.0",
     "semver-extra": "^3.0.0",
+    "shelljs": "^0.8.3",
```

Comment:
Can you help me understand why we need this dependency?

Reply:
It's required for spec files, where we use it to switch directories.